src/devices/bus/sega8/sega8_slot.cpp

```diff
@@ -941,6 +944,12 @@ void sms_cart(device_slot_interface &device)
 	device.option_add_internal("korean_nb",  SEGA8_ROM_KOREAN_NB);
 }
 
+void sms_card(device_slot_interface &device)
+{
+	device.option_add_internal("rom",  SEGA8_ROM_STD);
+	device.option_add_internal("3dadp",  SEGA8_ROM_3D_ADAPTOR);
```

Comment:
I don’t see why this option is internal – it should be directly accessible, since it isn’t used to load software.

Reply:
The internal option makes use of "-card" a requirement and this avoids loading two cards via the card slot at same time. Also see my reply to the previous question.